src/main/java/hu/blackbelt/structured/map/proxy/util/ReflectionUtil.java

```diff
@@ -36,6 +36,7 @@ public final class ReflectionUtil {
 
     private static final String GET_PREFIX = "get";
     private static final String SET_PREFIX = "set";
+    private static final String ADD_PREFIX = "addTo";
```

Comment:
What about "remove"?

Reply:
We discussed with @robertcsakany that addTo... is enough for the builder.